.eleventy.js

```diff
@@ -297,6 +297,12 @@ module.exports = function (config) {
     }
   }
 
+  // Chrometober config
+  config.addPassthroughCopy({
+    'src/site/content/en/chrometober-2022/': 'chrometober-2022',
```

Comment:
You don't want this to be copied over (it is an .njk file)

Reply:
It generates the HTML file in the given directory, no? As in `src/site/content/en/chrometober-2022/` creates an `index.html` at `chrometober-2022` which is what we want?